lexicons/app/bsky/feed/post.json

```diff
@@ -46,6 +47,27 @@
         "start": {"type": "integer", "minimum": 0},
         "end": {"type": "integer", "minimum": 0}
       }
+    },
+    "images": {
+      "type": "object",
+      "required": ["embed"],
+      "properties": {
+        "embed": {
```

Comment:
Could we call this attribute `images` instead of `embed`?

Reply:
Yeah I'm down with this. That's how I had it originally, but then thought it may be nice to keep the symmetry on the property value & just discriminate based on type 🤷‍♂️